Fiodor Borisiuk/0/run.rb

```diff
@@ -0,0 +1,132 @@
+require 'roo'
+
+file_list = Dir.entries('.')
```

Comment:
How did you download `.xslx` files? Where is script?

Reply:
added script for downloading .xlsx and .xls files to main